src/components/molecules/ArticleCard.tsx

```diff
@@ -9,16 +9,28 @@ export interface ArticleCardProps {
   title: string
   link: string
   writer?: string
+  clelabPick: boolean
```

Comment:
```suggestion
  badge: ReactNode
```
이면 더 유연한 설계가 될 것 같습니다

Reply:
da683c8d67c08000957dd3e686288962435fc549 수정했습니다 확인 부탁드립니다!